src/app/tests/suites/certification/Test_TC_FLW_1_1.yaml

```diff
@@ -38,8 +37,8 @@ tests:
           error: 1
 
     - label: "reads back global attribute: ClusterRevision"
-      command: "readAttribute"
       disabled: true
+      command: "readAttribute"
```

Comment:
why this move? was this supposed to be enabled now?

Reply:
No , we will enable it after this issue is fixed .https://github.com/project-chip/connectedhomeip/issues/9774